scripts/qa_crowbarsetup.sh

```diff
@@ -2614,7 +2614,7 @@ function custom_configuration()
         ceilometer)
             local ceilometerservice="ceilometer-cagent"
             if iscloudver 6plus ; then
-                ceilometerservice="ceilometer-polling"
+                ceilometerservice="ceilometer-central"
```

Comment:
So technically, this should not be done for cloudsource=GM6. This might matter for maintenance QA.